project-rome-docs/objectivec-api/connecteddevices/MCDConnectedDevicesNotificationRegistration.md

```diff
@@ -0,0 +1,35 @@
+---
+title: MCDConnectedDevicesNotificationRegistration
+description: This class represents the app's registration with a push notification service (necessary for some Project Rome scenarios). 
+keywords: microsoft, windows, iOS, iPhone, objectiveC, connected devices, Project Rome
+---
+
+# class `MCDConnectedDevicesNotificationRegistration` 
+
+```
+@interface MCDConnectedDevicesNotificationRegistration : NSObject
+```  
+ This class represents the app's registration with a push notification service (necessary for some Project Rome scenarios). It conveys this information to the Connected Devices Platform.
+
+## Properties
+
+### type
+`@property(nonatomic, readwrite) MCDConnectedDevicesNotificationType type;`
+
+The type of notifications in this setup.
+
+### token
+`@property(nonatomic, readwrite, nonnull) NSString* token;`
+
+The registration token.
+
+### appId
+`@property(nonatomic, readwrite, nonnull) NSString* appId;`
+
+The app ID for push notification registration.
+
+### appDisplayName
+`@property(nonatomic, readwrite, nonnull) NSString* appDisplayName;`
```

Comment:
It's not totally clear what "token" and "appId" mean here, so it would be good to link to the how-to page that describes how to populate these fields.

The how-to page looks like it's under construction right now.